dump-extensions/openpower-dumps/dump-extensions.cpp

```diff
@@ -42,6 +42,21 @@ void loadExtensions(sdbusplus::bus::bus& bus,
 
     dumpList.push_back(std::make_unique<openpower::dump::system::Manager>(
         bus, SYSTEM_DUMP_OBJPATH, SYSTEM_DUMP_OBJ_ENTRY));
+
```

Comment:
How you are handling the patch?

Looks, Change-Id is missed in the commit message, so we cannot track the Gerrit patch by using the commit message. 

FYI, You need to cherry-pick your upstream patch to downstream (you can get the command from the Gerrit patch itself by using the `Download` option).

Reply:
+1